docs/src/use/configure/configuration-files.md

```diff
@@ -495,3 +498,74 @@ npx eslint --config some-other-file.js **/*.js
 ```
 
 In this case, ESLint does not search for `eslint.config.js` and instead uses `some-other-file.js`.
+
+## TypeScript Configuration Files
+
+::: warning
+This feature is currently experimental and may change in future versions.
```

Comment:
The documentation should also explain how to enable this feature using `--flags`.